patterns/2-structured/innersource-portal.md

```diff
@@ -52,10 +52,32 @@ Key properties of the portal are:
 
 When launching the portal, a communications campaign promoting the addition of InnerSource data files or meta-data to code repositories should be considered, to bolster the number of projects displayed within the portal.
 
+### Implementations
+
+#### SAP Project Portal
+
 A [reference implementation](https://github.com/SAP/project-portal-for-innersource) of an InnerSource portal is available on GitHub and open for contributions. It lists all InnerSource projects of an organization in an interactive and easy to use way. Projects can self-register using a dedicated GitHub topic and provide additional metadata.
 
 ![Example of an InnerSource Portal](../../assets/img/portal-overview.png "Example of an InnerSource Portal")
 
+#### Wiki
```

Comment:
@rrrutledge I am assuming that this page is manually maintained?
i.e. teams would just come here and add a new row to the table?

Would it be worthwhile to describe somewhere at which point an org might outgrow this solution?
What the benefits of the manual approach are (to get started), and at which point that benefit turns into a problem?

There is room here to elaborate a bit on these points, to help people reason about which implementation to choose for their purposes. Doing this can be really hard, I know, especially for us folks that have looked at this problem space for so long already :)

Reply:
Good point - will try to have another “go” to incorporate this feedback.

Reply:
OK - added some text.

Reply:
Looks good!